apps/full-site-editing/full-site-editing-plugin/full-site-editing-plugin.php

```diff
@@ -98,7 +98,7 @@ function a8c_fse_on_plugin_activation() {
 	/**
 	 * Can be used to disable Full Site Editing functionality.
 	 *
-	 * @since 0.1
+	 * @since 0.2
```

Comment:
Was this an intentional change in this PR?

Reply:
It was. The filter was introduced in 0.2